.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
+   // Use IntelliSense to find out which attributes exist for C# debugging
```

Comment:
Not sure how I feel about this, but I guess there isn't a better option.

Reply:
Yeah I would like the comments in and VSCode add some of them as boilerplate and it seems to ignore them when parsing